openmls/src/ciphersuite/tests/mod.rs

```diff
@@ -0,0 +1,4 @@
+//! Unit tests for the ciphersuites.
```

Comment:
To make it easier to find files I generally prefer not to use too many `mod.rs`. What do you think about putting this in `ciphersuite/tests.rs` instead? (Similarly for all the others.)

I didn't touch any of this sort of re-org yet, but while you're at it we can at least change the ones we touch.

Reply:
I had thought about this before, my editor helpfully puts the directory behind the `mod.rs`, so it didn't impact me too much. But sure, let's start here and change the file names as we go from now on.

Reply:
Although the `mod.rs` approach does have the added benefit of making the relevant "mod" file easily identifiable in a given directory. I guess I don't have a strong opinion either way.